solidity/test/random_beacon_service/TestRelayRequestCallback.js

```diff
@@ -128,7 +128,7 @@ describe('KeepRandomBeacon/RelayRequestCallback', function() {
     let beneficiaryStartBalance = web3.utils.toBN(await web3.eth.getBalance(beneficiary));
 
     // use the same gas price as the gas price ceiling
-    let relayEntryTxGasPrice = web3.utils.toBN(web3.utils.toWei('30', 'Gwei'));
+    let relayEntryTxGasPrice = web3.utils.toBN(web3.utils.toWei('60', 'Gwei'));
```

Comment:
What do you think about reading `gasPriceCeiling` from the contract, defining it in `before` block and then reuse it in the tests? If later on we decide to change `gasPriceCeiling`, we won't need to update it, only gas prices around it.
```
let gasPriceCeiling;
before(async () => {
...
gasPriceCeiling = await operatorContract.gasPriceCeiling()
...
}

exampleTest {
// use the same gas price as the gas price ceiling
let relayEntryTxGasPrice = web3.utils.toBN(gasPriceCeiling);
}
```

Reply:
I was thinking about it but at the same time, I like the tests are checking a particular value, especially that we have tests using a higher and lower value. If someone makes a mistake and change value in the contract accidentally, tests will fail - I like it. This way we are asserting a specific expected behavior.

Reply:
right, makes sense.